infogami/utils/macro.py

```diff
@@ -3,11 +3,11 @@
 
 Macros take argument string as input and returns result as markdown text.
 """
-from __future__ import print_function
```

Comment:
This directive changes the Python 2 interpreter to make legacy print statements a syntax error on Python 2.  This keeps legacy developers from breaking Python 3 code.

Reply:
I'll add this back. I already went through and fixed the modules broken by 2to3 which use print("truc",file=machin), which breaks under Python 2, but missed this one because it didn't cause an error.